sentry-core/src/main/java/io/sentry/core/SentryEvent.java

```diff
@@ -67,7 +67,8 @@ public Date getTimestamp() {
   }
 
   @Nullable
-  Throwable getThrowable() {
+  @ApiStatus.Internal
+  public Throwable getThrowable() {
```

Comment:
I remember @bruno-garcia being against this, but if it's necessary, I'm fine with it.

Reply:
Otherwise I am not able to test it without reflection.

Reply:
we can't escape having this but the `@Internal` should do